static_src/components/panel_group.jsx

```diff
@@ -13,10 +20,14 @@ export default class PanelGroup extends React.Component {
   }
 
   render() {
+    const gridClass = `grid-width-${this.props.columns}`;
```

Comment:
Should this be changed like panel_block and default to 0 when columns not passed in, so you don't set any grid width class when you don't pass in columns number?